README.md

```diff
@@ -1,2 +1,102 @@
-# ES6_Awesome_Books
-Creation Book Library with ES6 Features
+# Awesome_Books
+Creation Book List Library for a user to store the list of books 
+
+> A demo website for Book Library.
+
+## Built With
+
+HTML and CSS
+Javascript : Arrays and Objects , DOM , Local Storage etc 
+
+## Video
+
+In the following [link](Coming Soon)
+
+
+## Live Server
+
+In the following link, you can view (Coming Soon)).
+
+## Screenshot
+
+![Book_Store_App(Page1)](./images/Image1.png)
+![Book_Store_App(Page2)](./images/Image2.png)
+![Book_Store_App(Page3)](./images/Image3.png)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+## Prerequisites
+
+### Browser
+
+Any modern browser works! :smile:
+
+
+### Knowledge
+
+Basic knowledge of the terminal and git
+
+
+### Getting Started
```

Comment:
- [ ] In the Readme file you have 2 `Getting Started` sections kindly remove one of them because this may confuse future users of your project